### PR TITLE
[Skale Changes] Changed Block.number to Block.timestamp

### PR DESCRIPTION
**Pull request by adi44**
For skale, changed block.number to block.timestamp. State and epoch calculation is dependent on block.timestamp now.

### CODE REVIEW

File: contracts/Core/StakeManager.sol [hunk 1]
```diff
@@ -296,9 +297,10 @@ contract StakeManager is Initializable, StakeStorage, StateManager, Pause, Stake
     function unstake(uint32 stakerId, uint256 sAmount) external initialized whenNotPaused {
         require(sAmount > 0, "Non-Positive Amount");
         require(stakerId != 0, "staker.id = 0");
+        // slither-disable-next-line timestamp
```
Comment (abhishekvispute) [inline contracts/Core/StakeManager.sol:300]: hmm how stakers stake is related to timestamp 
slither is suggesting issue here as well ?

**Review by abhishekvispute — APPROVED** [review]

File: contracts/Core/StakeManager.sol (hunk 1 above)
Comment (adi44) [reply contracts/Core/StakeManager.sol:300]: yeah it was suggesting here also

Comment (adi44) [reply contracts/Core/StakeManager.sol:300]: we can go through slither warnings of this PR tomorrow while we discuss other slither warnings

Comment (abhishekvispute) [reply contracts/Core/StakeManager.sol:300]: hmm cool

**Review by SamAg19 — APPROVED** [review]

**Pull request merged**